components/miller/stories/template.js

```diff
@@ -2,19 +2,35 @@ import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
 // import { ifDefined } from 'lit-html/directives/if-defined.js';
 
+import { Template as AssetList } from '@spectrum-css/assetlist/stories/template.js';
+
 import "../index.css";
 
 export const Template = ({
-  rootClass = "spectrum-Miller",
+  rootClass = "spectrum-MillerColumns",
   customClasses = [],
-  size = "m",
-  // ...globals
+  columns,
+  ...globals
 }) => {
+  if (!columns) return html``;
+
   return html`
-    <div class=${classMap({
-      [rootClass]: true,
-      [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
-      ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
-  })}></div>
+    <div
+      class=${classMap({
+        [rootClass]: true,
+        ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+      })}
+    >
+      ${columns.map(({items}) => {
+        return html`
+          <div class="${rootClass}-item">
+            ${AssetList({
+              items,
+              ...globals,
```

Comment:
I recommend passing in globals first so that if it contains any properties that conflict with your manually passed in values, yours will "win"